apollo-mockserver/pom.xml

```diff
@@ -47,7 +47,7 @@
     </dependency>
     <dependency>
       <groupId>org.springframework.boot</groupId>
-      <artifactId>spring-boot-starter</artifactId>
+      <artifactId>spring-boot-starter-test</artifactId>
```

Comment:
looks like it's not necessary as `spring-boot-starter-test` is included as a test scope dependency in [the parent pom](https://github.com/ctripcorp/apollo/blob/master/pom.xml#L354)